operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -380,7 +384,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
-	log.Info("Controller added")
+	scope.Info("Controller added.")
```

Comment:
by some poor grep, I think that <1% of logs are ending with a period today, nor are kubernetes or envoy logs. Should we be consistent?

Reply:
where things get weird is when we have multiple sentences. period after the first but not the last?
i agree about consistency but this one may be a bit of a grey area that i'd be happy to leave to individual discretion.